docs/consume-packages/Package-Source-Mapping.md

```diff
@@ -32,6 +41,41 @@ For a video-based overview of the Package Source Mapping feature, consider watch
 
 To opt into this feature, you must have a `nuget.config` file. Having a single `nuget.config` at the root of your repository is considered a best practice. See [nuget.config documentation](../reference/nuget-config-file.md) to learn more.
 
+### Enable by using Visual Studio Options Dialog
+
+1. Open your solution in Visual Studio.
+2. Navigate to the `Package Source Mappings` Options Dialog.
+
+_From the Package Manager UI_
+
+- Select a package from the list to show it in the Details Pane.
+- Press the `Configure` button to open the Package Source Mappings options page.
+
+![The NuGet Package Manager window in Visual Studio showing a selected package, and a highlight around the "Package source mapping is off" status with a `Configure` button.](media/packageSourceMapping_PMUI_Status_Off_Annotated.png)
+
+_From the Visual Studio Options Dialog_

```

Comment:
This seems a bit confusing. Should we show this selection of Tools? Or could someone just open generic VS Options too?

Reply:
Yes, my point was that you can either use Tools menu or you can use PM UI with the `Configure` shortcut.
I can show it... but I thought selecting Tools was convered in another docs.